frontend/include/chpl/parsing/parsing-queries.h

```diff
@@ -616,6 +616,15 @@ ID idToContainingMultiDeclId(Context* context, ID id);
 bool idContainsFieldWithName(Context* context, ID typeDeclId,
                              UniqueString fieldName);
 
+/**
+  Given an AST node for a (multi-)declaration, find for a Variable
+  node that declares a field of the given name. Returns whether the field
+  was found, and, if it was, sets outFieldId to the ID of the Variable.
+ */
+bool findFieldIdInDeclaration(const uast::AstNode* varDecl,
+                              UniqueString fieldName,
+                              ID& outFieldId);
+
 /**
   Given an ID for a Record/Union/Class Decl,
   and a field name, returns the ID for the Variable declaring that field.
```

Comment:
Could you update this (or somewhere else if you think there's a better place) to mention that this (and related functions) look for fields with that name:
 * directly declared in the class/record with `const` `var` `param` `type`
 * declared within a multi decl (`var x, y, z: t`)
 * declared within a tuple decl (`var (x, y, z): 3*t`)
 * declared within a forwarding statement (`forwarding var x: t`)